### PR TITLE
add setup.py

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    with open(ROOT + '/' + filename) as requirements_file:
+        install_requires = requirements_file.read().splitlines()
+    return install_requires
+
+
+setup(
+    name=NAME,
+    version=VERSION,
+    author='UNICEF',
+    author_email='rapidpro@unicef.org',
```
Comment (czue) [inline setup.py]: is this a valid email?

File: data_api/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+NAME = 'rapidpro-dw'
```
Comment (czue) [inline data_api/__init__.py]: thoughts on extending to `'rapidpro-data-warehouse'`? i'm generally acronym averse...

File: setup.py (hunk 1 above)
Comment (domdinicola) [reply setup.py]: do you have a valid email?

File: data_api/__init__.py (hunk 2 above)
Comment (domdinicola) [reply data_api/__init__.py]: sure thing

File: setup.py (hunk 1 above)
Comment (czue) [reply setup.py]: I think this is good actually. I believe it's a shared account evan can access.

**Pull request merged**